src/Models/Organization.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class Organization extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        "name",
+        "description",
+        "location",
+        "organization_type",
+        "foundation_date",
+        "number_of_employers",
+        "website_url",
+        "facebook_url",
+        "linkedin_url",
+        "instagram_url",
```

Comment:
Maybe create trait like this - https://github.com/brewmap/brewmap/blob/develop/app/Eloquent/Traits/CamelCaseAttributes.php

Reply:
And you could use `instagramUrl` instead of this `instagram_url` - `$fillable` array will stay like now but in other clasess when u are using `organization` class, u could use camel case (`$organization->instagramUrl`).